src/content/app/tools/blast/components/blast-app-bar/BlastAppBar.test.tsx

```diff
@@ -49,14 +52,16 @@ const mockCommittedItems = [
 
 const initialState = {
   blast: { blastForm: initialBlastFormState },
+  general: { view: initialBlastGeneralState },
```

Comment:
This isn't in the right place

Reply:
good point. Not sure why i added this here but i remember i had to add the blastGeneralReducer and looks like this is enough, it doesnt require the state. I have updated the code.

Reply:
I mean if the test doesn't require it (is doesn't seem to; it wasn't complaining when the state was set up incorrectly), then don't include it. Otherwise it makes an impression that this piece of state is required for the test to run.

Reply:
But if i removed the blastGeneralReducer then the test fails. So i left the reducer in there. And i thought if you are adding the reducer, you also need to add the state but thats not the case.

Reply:
Oh, ok then.